straxen/plugins/event_info.py

```diff
@@ -23,11 +23,12 @@ class EventInfo(strax.MergeOnlyPlugin):
                   'event_positions',
                   'corrected_areas',
                   'energy_estimates',
+                  'event_w_bayes_class',
                   # 'event_pattern_fit', <- this will be added soon
                   ]
     save_when = strax.SaveWhen.ALWAYS
     provides = 'event_info'
-    __version__ = '0.0.2'
+    __version__ = '0.0.3'
```

Comment:
Can we exclude it from event_info like we do for `event_pattern_fit`. Then we can process it fairly independently from the main data and hence performance is less of an issue if things arrive late.

Reply:
I will leave this up to you